tests/Composer/Installers/Test/CakePHPInstallerTest.php

```diff
@@ -63,9 +63,20 @@ public function testInflectPackageVars()
     public function testGetLocations() {
         $package = new RootPackage('CamelCased', '1.0', '1.0');
         $composer = $this->composer;
+
+        $io = $this->getMock('Composer\IO\IOInterface');
+        $config = $this->getMock('Composer\Config');
+
+        // Simultaneous support for Composer 1 and 2
```

Comment:
The test also directly touches Composer APIs that changed in 2.0; use Reflection to do the right thing